docs/installing-solidity.rst

```diff
@@ -335,8 +335,8 @@ Minimum compiler versions
 
 The following C++ compilers and their minimum versions can build the Solidity codebase:
 
-- `GCC <https://gcc.gnu.org>`_, version 5+
-- `Clang <https://clang.llvm.org/>`_, version 3.4+
+- `GCC <https://gcc.gnu.org>`_, version 9+
+- `Clang <https://clang.llvm.org/>`_, version 10+
```

Comment:
These also haven't been updated in a while it seems :-). If we go for this PR, I suggest to bump to the versions available on Ubuntu 20.04 focal: see https://packages.ubuntu.com/focal/gcc and https://packages.ubuntu.com/focal/clang

Reply:
So what did we decide to use on the call today? GCC 8.3?

Reply:
I'll change it to whatever major version Debian Stable has, that's less than what ubuntu 20.04 will have, but should be fine. I'll also just update the CMake checks to actually check for these here.